src/diffusers/models/modeling_utils.py

```diff
@@ -377,6 +377,11 @@ def save_pretrained(
         logger.info(f"Model weights saved in {os.path.join(save_directory, weights_name)}")
 
         if push_to_hub:
+            # Create a new empty model card and eventually tag it
+            model_card = load_or_create_model_card(repo_id=repo_id, token=token)
```

Comment:
```suggestion
            model_card = load_or_create_model_card(repo_id, token=token)
```

(if arg is renamed -see comment below-)